meta-resin-common/classes/image-resin.bbclass

```diff
@@ -0,0 +1,23 @@
+#
+# Resin images customizations
+#
+
+# Deploy license.manifest
+DEPLOY_IMAGE_LICENSE_MANIFEST = "${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.license.manifest"
+DEPLOY_SYMLINK_IMAGE_LICENSE_MANIFEST = "${DEPLOY_DIR_IMAGE}/${IMAGE_LINK_NAME}.license.manifest"
+IMAGE_LICENSE_MANIFEST = "${LICENSE_DIRECTORY}/${IMAGE_NAME}/license.manifest"
+# Deplot the license.manifest of the current image we baked
+deploy_image_license_manifest () {
+    cp -f ${IMAGE_LICENSE_MANIFEST} ${DEPLOY_IMAGE_LICENSE_MANIFEST}
+    ln -sf ${IMAGE_NAME}.rootfs.license.manifest ${DEPLOY_SYMLINK_IMAGE_LICENSE_MANIFEST}
```

Comment:
Guess you can have this changed to:

ln -sf ${DEPLOY_IMAGE_LICENSE_MANIFEST} ${DEPLOY_SYMLINK_IMAGE_LICENSE_MANIFEST}